src/pages/index.js

```diff
@@ -92,7 +210,29 @@ addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
 
+// add event listener to avatar
+document
+  .querySelector(".profile__avatar-edit")
+  .addEventListener("click", () => {
+    editAvatarModal.open();
+  });
+// when it fires, open avatar image modal
+
+const editFormValidator = new FormValidator(settings, profileEditForm);
+const addFormValidator = new FormValidator(settings, addCardFormElement);
+const editAvatarValidator = new FormValidator(settings, editAvatar);
+
 editFormValidator.enableValidation();
 addFormValidator.enableValidation();
+editAvatarValidator.enableValidation();
+
+api.getInitialCards().then((res) => {
+  console.log(res);
+  cardList.addItem(createCard(res[0]));
```

Comment:
`  cardList.addItem(createCard(res[0]));
`
* When I reload the page, all my new cards disappear


`renderItems` should be used here to render the array of cards